stores/metadata_test.go

```diff
@@ -1447,6 +1447,9 @@ func TestObjectEntries(t *testing.T) {
 
 		{"/", "", "health", "ASC", []api.ObjectMetadata{{Name: "/foo/", Size: 10, Health: .5}, {Name: "/FOO/", Size: 7, Health: 1}, {Name: "/fileś/", Size: 6, Health: 1}, {Name: "/gab/", Size: 5, Health: 1}}},
 		{"/", "", "health", "DESC", []api.ObjectMetadata{{Name: "/FOO/", Size: 7, Health: 1}, {Name: "/fileś/", Size: 6, Health: 1}, {Name: "/gab/", Size: 5, Health: 1}, {Name: "/foo/", Size: 10, Health: .5}}},
+
+		{"/", "", "size", "DESC", []api.ObjectMetadata{{Name: "/foo/", Size: 10, Health: .5}, {Name: "/FOO/", Size: 7, Health: 1}, {Name: "/fileś/", Size: 6, Health: 1}, {Name: "/gab/", Size: 5, Health: 1}}},
```

Comment:
Maybe we should extend `GetObjectOptions` with `sortBy` and `sortDir` so you can use the bus client and effectively pass these sorting parameters, currently not the case. I noticed because I was going to `nit` here and ask you to extend the integration test to verify the sorting params get parsed properly...